### PR TITLE
Added: `selector-max-id` rule.

### PR DESCRIPTION
**Pull request by evilebottnawi**
> Which issue, if any, is this issue related to?

#2528

> Is there anything in the PR that needs further explanation?

No, it's self explanatory.

### CODE REVIEW

File: lib/rules/selector-max-id/__tests__/index.js [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+
+  reject: [ {
+    code: "#foo #bar #baz {}",
+    description: "compound selector: greater than max id selectors",
+    message: messages.expected("#foo #bar #baz", 2),
+    line: 1,
+    column: 1,
+  }, {
+    code: "#foo, \n#bar #baz #foo {}",
+    description: "multiple selectors: greater than max classes",
```
Comment (hudochenkov) [inline lib/rules/selector-max-id/__tests__/index.js]: "multiple selectors: greater than max id selectors"

**Comment by jeddy3** [comment]
@hudochenkov I've made the changes. Do they look good to you?

**Comment by hudochenkov** [comment]
@jeddy3 looks good!

**Comment by jeddy3** [comment]
Changelog:

> -   Added: `selector-max-id` rule ([#2654](https://github.com/stylelint/stylelint/pull/2654)).

**Pull request merged**